docs/src/css/custom.css

```diff
@@ -28,6 +28,6 @@
   --secondary-light: #ccc;
 }
 
-p a, .a {
+article a, .a {
```

Comment:
I would keep the "p a" in addition

Reply:
I think `article` is actually what we want? That encapsulates every document body. `p a` I think is much shakier